### DISCUSSION TITLE
How to ask for help?

### DISCUSSION THREAD

**Discussion by jcfrog** [aclap-dev/video-downloadhelper#3] [Help and Q&A] [answered]
Please read [these brief recommendations](https://github.com/aclap-dev/video-downloadhelper/wiki/I-need-help#start-a-new-qa-post) then [start your post](https://github.com/aclap-dev/video-downloadhelper/discussions/new?category=help-and-q-a) (you might want to [create a Github account first](https://github.com/signup)).

**Comment by needhelpjal** [comment]
Hi I use ubuntu studio 18.4 - video  download helper wants me to upgrade but how do i do the up grade in a terminal as using  the install manager, the install fails to install all apps and has never worked I can only install apps via the terminal

**Reply by paulrouget** [reply]
Download the [deb file](https://github.com/aclap-dev/vdhcoapp/releases/latest/download/vdhcoapp-linux-x86_64.deb) then in a terminal, go to your download directory, and do `sudo dpkg -i ./vdhcoapp-linux-x86_64.deb`.

If you have more questions, please [open a new issue](https://github.com/aclap-dev/video-downloadhelper/discussions/new?category=help-and-q-a).

**Comment by paulrouget** [comment, marked as answer]
> Please read [these brief recommendations](https://github.com/aclap-dev/video-downloadhelper/wiki/I-need-help#start-a-new-qa-post) then [start your post](https://github.com/aclap-dev/video-downloadhelper/discussions/new?category=help-and-q-a) (you might want to [create a Github account first](https://github.com/signup)).
